repos/system_upgrade/common/libraries/mounting.py

```diff
@@ -422,3 +422,22 @@ def _mount_options(self):
             '-t', 'overlay', 'overlay2',
             '-o', 'lowerdir={},upperdir={},workdir={}'.format(self.source, self._upper_dir, self._work_dir)
         ]
+
+
+def mount_upgrade_iso_to_root_dir(root_dir, target_iso):
+    """
+    Context manager mounting the target RHEL ISO into the system root residing at `root_dir`.
+
+    If the `target_iso` is None no action is performed.
+
+    :param root_dir: Path to a directory containing a system root.
+    :type root_dir: str
+    :param target_iso: Description of the ISO to be mounted.
+    :type target_iso: Optional[TargetOSInstallationImage]
+    :rtype: Optional[LoopMount]
+    """
+    if target_iso:
```

Comment:
Nit: again I'd invert check
```
if not target_iso:
    return
...
return LoopMount(..)
```

Reply:
Resolved by accepting the suggested modification.